### DISCUSSION TITLE
RDF diagram

### DISCUSSION THREAD

**Discussion by alib-l** [deepmodeling/deepmd-kit#677] [Q&A]
Hello,
I want to plot the radial distribution function for the deepmd to compare with DFT results.
Could you please help me what I have to do?
Thanks,

**Comment by tuoping** [comment]
This is an excellent work for your reference. Hope this helps you. 
https://www.sciencedirect.com/science/article/pii/S1005030220309075

**Reply by alib-l** [reply]
Thanks so much
